src/pages/AppList.svelte

```diff
@@ -57,7 +58,7 @@
 
   function getAppLogo(app) {
     if (app.match(/fenix/) || app.match(/nightly/)) {
-      return appLogos.nightly;
+      return appLogos.browser;
```

Comment:
We used to index Firefox Nightly but not anymore, so I think we can safely remove line 59. Since currently the only product that has "fenix" in its name is Firefox for Android, I think it might be better to add this condition to line 81, something like:

```
if (app.match(/firefox/) || app.match(/fenix/)) {
      return appLogos.browser;
}
```